src/Game.css

```diff
@@ -1,4 +1,35 @@
 .direction-buttons {
     display: inline-block;
     margin: 10px auto;
+}
+
+button {
```

Comment:
Don't use generic names. Rename this CSS component to something meaningful which this CSS style is trying to achieve.

Reply:
This is not generic. it selects every html button element